types/node-hid/index.d.ts

```diff
@@ -31,6 +31,6 @@ export class HID {
     resume(): void;
     on(event: string, handler: (value: any) => void): void;
     write(values: number[]): number;
-    setDriverType(type: string): void;
 }
 export function devices(): Device[];
+export function setDriverType(type: string): void;
```

Comment:
thx for the changes. I think the following solution is more type safe:
```diff
-export function setDriverType(type: string): void;
+export function setDriverType(type: 'hidraw' | 'libusb'): void;
```

Reply:
Good idea